cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -103,9 +101,7 @@ spec:
       - name: healthz
         image: gcr.io/google_containers/exechealthz-amd64:1.1
         resources:
-          # keep request = limit to keep this container in guaranteed class
           limits:
-            cpu: 10m
```

Comment:
Please add a comment saying that the pod is anyway in "burstable" because of memory, so cpu limits don't matter. Also include that we *don't* want to set cpu until we've profiled the sub components thoroughly, because on debian systems the enforcement is more strict.